my-world/app/routes/mcu/index.tsx

```
@@ -0,0 +1,181 @@
+import React from 'react'
+import type {Route} from './+types/index';
+import { Loading } from '~/components/loadingIcon';
+import MovieCard from '~/components/index_page/movie_card';
+import { HeroBanner } from '~/components/index_page/hero_banner';
+import WatchOrder from '~/components/watch_order/watchOrder';
+import { loadProgress } from '~/utils/db';
+
+export function meta({}: Route.MetaArgs ) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)